support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -0,0 +1,104 @@
+import { expect, test } from '@playwright/test';
+import { email, firstName, lastName } from './utils/users';
+import { setTestUserDetails } from './utils/testUserDetails';
+import { checkRecaptcha } from './utils/recaptcha';
+import { fillInCardDetails } from './utils/cardDetails';
+import { fillInDirectDebitDetails } from './utils/directDebitDetails';
+import { fillInPayPalDetails } from './utils/paypal';
+import { setupPage } from './utils/page';
+import { afterEachTasks } from './utils/afterEachTest';
+
+interface TestDetails {
+	paymentType: 'Credit/Debit card' | 'Direct debit' | 'PayPal';
+	tier: 1 | 2 | 3;
+	frequency: 'Monthly' | 'Annual';
+	country?: 'US' | 'AU';
+}
+
+const testsDetails: TestDetails[] = [
+	{ paymentType: 'Credit/Debit card', tier: 1, frequency: 'Monthly' },
+	{ paymentType: 'Credit/Debit card', tier: 2, frequency: 'Annual' },
+	{ paymentType: 'Direct debit', tier: 2, frequency: 'Monthly' },
```

Comment:
Do we need an `Annual` direct debit here?

Reply:
I don't think so, the functionality is exactly the same on Monthly w/ Direct Debit and Annual w/ Direct Debit across both tiers so I don't think the test would bring any extra benefit than what we have now.

Reply:
Makes sense, thanks